node-src/ui/messages/errors/sentinelFileErrors.ts

```diff
@@ -0,0 +1,9 @@
+import chalk from 'chalk';
+import { dedent } from 'ts-dedent';
+
+import { error } from '../../components/icons';
+
+export default () =>
+  dedent(chalk`
+    ${error} Failed to finalize upload. Please check https://status.chromatic.com/ or contact support.
```

Comment:
Did we want to turn this into a link like this?

https://github.com/chromaui/chromatic-cli/blob/9cd04adac330e2223f1d781c2f536ee20117ea19/node-src/ui/messages/errors/gitNotInitialized.ts#L21

It should just be a quick import

```js
import link from '../../components/link';
```

Reply:
Good thinking!